# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1422041863 18000
#      Fri Jan 23 14:37:43 2015 -0500
# Branch gui-release
# Node ID 4c1f638a381edc50aee0ced2ee5815a9e5c78218
# Parent  4c3ea57905fdc8abb00bb1af7bca27a705277d1c
avoid old-style cast warning in libgui

* graphics/Backend.cc: Replace C-style cast with reinterpret_cast.

diff --git a/libgui/graphics/Backend.cc b/libgui/graphics/Backend.cc
--- a/libgui/graphics/Backend.cc
+++ b/libgui/graphics/Backend.cc
@@ -93,17 +93,17 @@ bool Backend::initialize (const graphics
     {
       Logger::debug ("Backend::initialize %s from thread %08x",
 		     go.type ().c_str (), QThread::currentThreadId ());
 
       ObjectProxy* proxy = new ObjectProxy ();
       graphics_object gObj (go);
 
       gObj.get_properties ().set(toolkitObjectProperty (go),
-				 OCTAVE_PTR_TYPE ((OCTAVE_INTPTR_TYPE) proxy));
+				 OCTAVE_PTR_TYPE (reinterpret_cast <OCTAVE_INTPTR_TYPE> (proxy)));
 
       emit createObject (go.get_handle ().value ());
 
       return true;
     }
 
   return false;
 }
